tests/mock/AnnotatorPageTest.php

```diff
@@ -3,9 +3,12 @@
 namespace SilverLeague\IDEAnnotator\Tests;
 
 // Why is this required?
-// @todo get it to work properly
-require_once(BASE_PATH . '/mysite/code/Page.php');
-require_once(BASE_PATH . '/mysite/code/PageController.php');
+// @todo get it to work properly. Seems something is wrong with the installation of the CMS
+// We do need the CMS and the pages for proper testing
+if (!class_exists('\\Page')) {
+    require_once(BASE_PATH . '/mysite/code/Page.php');
```

Comment:
We still need to get this of this code - in SS 4.2 this is in `/app/src/Page.php` by default, but we should be able to rely on the PSR-0 autoloader definition. Unrelated to this PR though.

Also unrelated, but require_once is a construct rather than a function so doesn't need brackets

Reply:
I know it doesn't require brackets, and I agree it needs to go out.
But both are not high prio for me, unless actively developing on this module, it would not hurt anyone (yet).

The require_once not needing brackets is true, I guess it's a personal preference to use the brackets, for me that is. It just feels... "more right"